front/lib/plans/subscription.ts

```diff
@@ -307,7 +305,7 @@ export const pokeUpgradeWorkspaceToPlan = async (
   }
 
   const isUgradeToEnterprise = newPlan.code.startsWith("ENT_");
-  const isUpgradeToPro = isProPlanCode(newPlan.code);
+  const isUpgradeToPro = newPlan.code.startsWith("PRO_");
```

Comment:
Not perfect but that's only for upgrades from Poké! This is just some sanity checks for us same as we do for Enterprise plans to have a pit of success for ops.